otri/filtering/filters/summer_filter.py

```diff
@@ -10,21 +10,21 @@ class SummerFilter(Filter):
         Single stream
     '''
 
-    def __init__(self, input_stream: Stream, keys_to_change: Collection[str], const : int):
+    def __init__(self, input_stream: Stream, keys_to_change: Collection[str], constants : Mapping):
```

Comment:
Can directly be a mapping of the keys to their constants

Reply:
Sure